README.md

```diff
@@ -26,11 +26,18 @@ webrun ./example.js
 Then in your JS:
 
 ```javascript
+// Load code from an HTTPS server
 import example from "https://rangermauve.hashbase.io/esm.js";
-import p2pexample from "dat://rangermauve.hashbase.io/esm.js";
+
+// Load from the dat network
+import datExample from "dat://rangermauve.hashbase.io/esm.js";
+
+// Load from the IPFS network. Might not always be online.
```

Comment:
Why the "Might not always be online" warning here, and not for `dat`?

(I'm guessing it's that the IPFS ecosystem doesn't have a pinning service like Hashbase, but don't know enough about it to be sure.)

Reply:
I don't have the contents pinned personally, but if you know of a pinning service for IPFS I'll definitely add it there